src/task/CleanStoppedSessionTask.ts

```diff
@@ -41,6 +41,8 @@ export default class CleanStoppedSessionTask extends BaseTask {
       this.context.statsCollector = null;
       this.context.connectionMonitor.stop();
       this.context.connectionMonitor = null;
+      this.context.idleMonitor.removeIdleMonitorObserver();
```

Comment:
Here, once you changed them to `start()` and `stop()`, this will be `stop()` that matches other monitor components.